core/services/vrf/validate.go

```diff
@@ -15,7 +17,10 @@ var (
 )
 
 func ValidateVRFSpec(tomlString string) (job.Job, error) {
-	var jb = job.Job{Pipeline: *pipeline.NewTaskDAG()}
+	var jb = job.Job{
+		ExternalJobID: uuid.NewV4(), // Default to a random uuid
```

Comment:
Does it matter for validation?

Reply:
if we don't have a default here it will fail the not null db constraint